src/fielddef.ts

```diff
@@ -20,6 +20,8 @@ export interface FieldRefOption {
   binSuffix?: string;
   /** append suffix to the field ref (general) */
   suffix?: string;
+  /** return the base field, not rank */
+  noRank?: boolean;
```

Comment:
Hmm, the problem I see is that we have way more places where we want rank by default and only a handful of places where we do not want the rank.